AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -85,25 +87,98 @@ void ASDisplayNodeRespectThreadAffinityOfNode(ASDisplayNode *node, void (^block)
   }
 }
 
-+ (void)initialize
-{
-  if (self == [ASDisplayNode class]) {
-    return;
+/**
+ *  Returns ASDisplayNodeFlags for the givern class/instance. instance MAY BE NIL.
+ *
+ *  @param c        the class, required
+ *  @param instance the instance, which may be nil. (If so, the class is inspected instead)
```

Comment:
It is unclear to me why the instance parameter is ever desired.  Can you remove it, or update the comment to be clearer on why it exists?  Since the class is required and the behavior seems to be correct in the case that no instance is provided, why would we ever provide an instance?  Namely, -instancesRespondToSelector: seems to be sufficient.